OptimizelySDK/Optimizely.cs

```diff
@@ -308,5 +296,226 @@ public Variation GetForcedVariation(string experimentKey, string userId)
 
             return forcedVariation;
         }
+
+        #region  FeatureFlag APIs
+
+        /// <summary>
+        /// Determine whether a feature is enabled.
+        /// Send an impression event if the user is bucketed into an experiment using the feature.
+        /// </summary>
+        /// <param name="experimentKey">The experiment key</param>
+        /// <param name="userId">The user ID</param>
+        /// <param name="userAttributes">The user's attributes.</param>
+        /// <returns>True if feature is enabled, false otherwise</returns>
+        public bool IsFeatureEnabled(string featureKey, string userId, UserAttributes userAttributes = null)
+        {
+            if (string.IsNullOrEmpty(userId))
+            {
+                Logger.Log(LogLevel.ERROR, "User ID must not be empty.");
+                return false;
```

Comment:
return null. See doc

Reply:
Java has a different implementation in these cases. Anyways, following the doc.